source/gameengine/Converter/KX_ConvertActuators.cpp

```diff
@@ -977,7 +977,7 @@ void BL_ConvertActuators(const char* maggiename,
 			}
 
 			tmp = new SCA_2DFilterActuator(gameobj, filtermode,  _2dfilter->flag,
-			                               _2dfilter->float_arg, _2dfilter->int_arg,
+			                               _2dfilter->float_arg, _2dfilter->int_arg, _2dfilter->mipmap,
```

Comment:
What i mean is that you could let the conversion from int to bool only in this file. And so use bool everywhere in SCA_2DFilterActuator and KX_2DFilterData.